### ISSUE TITLE
Decision about public video recordings

### ISSUE THREAD

**Issue by codeboten** [open-telemetry/community#1406]
There is a growing concern that the meeting recordings could be used to create impersonations of community members. Some members have asked that we stop publishing public recordings.

The following options were proposed in the maintainers SIG call and a vote the preferred option was suggested:

👍 Leave the recordings up (accept the risk)
😄 Flag the videos as not to be used in training data (wouldn’t protect against targeted attacks, but still helpful)
🎉 Make the recordings private to the community (unlisted on YouTube by default, accessible via playlist or links in a doc)
❤️ Make the recordings expire or become unlisted after some time period (or only accessible to GC / admins eventually)
🚀 Rely on transcripts
👀 Rely on manual notes

More information as per @dyladan's comment below:

> There are several other factors at play here aside from the preferences of the people voting on github. Specifically, we may have legal obligations relating to the right to be forgotten in certain jurisdictions. I'll reach out to the CNCF to see if they can provide guidance in this area. The people voting here are not a representative sample of affected people. Most people who join the meetings are not maintainers and are unlikely to see community issues like this one. Most of them are also not public personas and they may not be as comfortable with public recordings of themselves as others who are more used to being public. As mentioned by others, some changes may also require charter changes which falls under the purview of the GC
> 
> We should also consider that the solution may involve aspects of multiple options in the poll. I think nearly everyone agrees some expiration/retention policy is an improvement on what we already have. We know from the maintainers meeting that some people are definitely using these recordings and we don't want to exclude them from the project, but most of the usefulness of the recordings is lost after a short period of time. We need to make sure that we're balancing the need for privacy with the usefulness of the public recordings. I think most would also agree that better meeting notes would be an improvement for everyone, not just people who can't attend meetings.

**_It is okay to vote for multiple options_**.

**Comment by austinlparker** [comment]
Quick clarifying question - is this concern specifically about video, or is it about combined audio/video, or all of the above?

**Comment by codeboten** [comment]
> Quick clarifying question - is this concern specifically about video, or is it about combined audio/video, or all of the above?

Based on the conversation i believe it was both

**Comment by austinlparker** [comment]
FWIW, I believe any change to completely remove recordings will require changes to the charter. I would also advise that auto-generated transcripts are not sufficient in terms of accessibility for meeting recordings, imo.

**Comment by Aneurysm9** [comment]
> I would also advise that auto-generated transcripts are not sufficient in terms of accessibility for meeting recordings, imo.

In addition to the accessibility concerns of reliance on auto-generated transcripts I'm not comfortable with the risk of mistranscription attributing statements to people that they did not make.

**Comment by austinlparker** [comment]
Yeah basically the option isn't "recordings or transcripts", it's "recordings or manual notes only". I'd remove that option from the poll tbh.

**Comment by tigrannajaryan** [comment]
> Yeah basically the option isn't "recordings or transcripts", it's "recordings or manual notes only". I'd remove that option from the poll tbh.

@codeboten can you add "Manual note only" as an option to vote for? And perhaps we keep the "record transcripts" as an option to vote too since we already have several votes (not dismissing @Aneurysm9's concern about misattribution, I think the concern is very valid).

Anyone who already voted for "transcripts" but believes "manual notes" are preferable please re-cast your vote.

**Comment by codeboten** [comment]
@tigrannajaryan done, i've added manual notes.

@plantfansam @jpkrohling @mx-psi Please review the comments here to see if you'd like to change your vote to manual notes.

**Comment by Aneurysm9** [comment]
I think for manual notes to replace recordings we need to establish a policy and provide tooling to enable consistency and completeness in the notes, similar to what the W3C does with their [meeting policy](https://www.w3.org/2021/Process-20211102/#GeneralMeetings) and [scribe system](https://www.w3.org/2008/04/scribe.html).

**Comment by dyladan** [comment]
Thanks @codeboten for creating this issue after the meeting. There are several other factors at play here aside from the preferences of the people voting on github. Specifically, we may have legal obligations relating to the right to be forgotten in certain jurisdictions. I'll reach out to the CNCF to see if they can provide guidance in this area. The people voting here are not a representative sample of affected people. Most people who join the meetings are not maintainers and are unlikely to see community issues like this one. Most of them are also not public personas and they may not be as comfortable with public recordings of themselves as others who are more used to being public. As mentioned by others, some changes may also require charter changes which falls under the purview of the GC

We should also consider that the solution may involve aspects of multiple options in the poll. I think nearly everyone agrees some expiration/retention policy is an improvement on what we already have. We know from the maintainers meeting that some people are definitely using these recordings and we don't want to exclude them from the project, but most of the usefulness of the recordings is lost after a short period of time. We need to make sure that we're balancing the need for privacy with the usefulness of the public recordings. I think most would also agree that better meeting notes would be an improvement for everyone, not just people who can't attend meetings.

**Comment by dyladan** [comment]
> @codeboten I assumed when voting that it is okay for a single person to vote for multiple options, but I think it would be good to explicitly state this (or deny it) on the issue text so that people are aware of the rules

The purpose of this is to see what the preferences of the community are, but it is not a binding vote. The GC is going to review the options and available information and decide on a course forward. Selecting multiple options should be fine because it gives the GC the most information for when it makes a final decision.

**Comment by tylerbenson** [comment]
I'm strongly in favor of replacing the videos with auto generated transcripts.  I understand the concern about mis-attributing statements, but that same problem can exist with poorly taken notes and I think that can be minimized by making sure it is clear these are computer generated transcripts.  It would be far more consistent across sig meetings vs relying on individual note takers.  Some meetings have great note takers, others do not.

I find the current system of recording is broken and mostly unusable for regular follow up due to this problem:
https://github.com/open-telemetry/community/issues/715#issuecomment-1376455525

**Comment by weyert** [comment]
What happens when there would be a dispute about the meeting minutes? Also how well does auto generated transcripts work for people that don't speak BBC English?

How can a person not part of a SIG validate the meeting minutes are correct?

**Comment by trask** [comment]
I personally see GitHub issues and PRs as the only source of truth. And I see meetings as just there to help establish consensus and move GitHub issues and PRs forward.

I'm curious if we said this explicitly as a project, would that alleviate concerns around inaccurate transcripts and disputed meeting minutes?

**Comment by trask** [comment]
> I believe any change to completely remove recordings will require changes to the charter. I would also advise that auto-generated transcripts are not sufficient in terms of accessibility for meeting recordings, imo

@austinlparker can you elaborate on both of these points? thx!

**Comment by austinlparker** [comment]
> > I believe any change to completely remove recordings will require changes to the charter. I would also advise that auto-generated transcripts are not sufficient in terms of accessibility for meeting recordings, imo
> 
> @austinlparker can you elaborate on both of these points? thx!

Sure. We specifically call out recordings of meetings under the communication policy/guidelines doc:
https://github.com/open-telemetry/community/blob/169df8372157e0cad5360958c4f883e81f128890/COMMUNICATION.md?plain=1#L47

In terms of accessibility, there are several blogs and other resources about automatic transcription - one such is https://www.boia.org/blog/do-youtubes-automatic-captions-improve-accessibility, or https://cdl.ucf.edu/why-automatic-captions-are-not-enough

tl;dr, any automatic transcription software would need to be 100% accurate, which they aren't, in order to be accessible. Now, I'm not saying that we actually do that good at accessibility to begin with -- it's not a specific goal of the meeting recordings, after all, they're more about ensuring _access_ -- but the rationale behind why auto-transcripts aren't accessible applies here as well. In fact, I'd argue that auto-transcripts would be worse in this case because there's a high chance that they spit out incorrect statements, which then gets hoovered up into GPT-whateverthefuck and now the LLMs are giving even more confidently wrong answers.

Either way, this is why I don't really feel like machine-generated transcripts are a viable replacement for audio or video recordings from an access or an accessibility angle.

**Comment by tylerbenson** [comment]
I agree with @trask that the video/audio/transcription should not be our source of truth.

I disagree that transcription would need 100% accuracy.  If we are relying on people taking notes, those certainly don't have that level of accuracy. We can't even attribute note taking because everyone is anonymous and writable by everyone, so a malicious actor could easily modify the notes to say whatever they want.  I think the notes are a useful tool for discussion during the meeting and a nice overview afterwards, but should not be considered the source of truth.

Especially if we choose to not make recordings available, I feel auto generated transcripts would on average be more helpful to understand what was discussed in a meeting than the notes alone.

**Comment by austinlparker** [comment]
> I disagree that transcription would need 100% accuracy. If we are relying on people taking notes, those certainly don't have that level of accuracy. We can't even attribute note taking because everyone is anonymous and writable by everyone, so a malicious actor could easily modify the notes to say whatever they want. I think the notes are a useful tool for discussion during the meeting and a nice overview afterwards, but should not be considered the source of truth.

The fundamental difference between imperfect human notes and imperfect computer-generated transcription is the _human_ part. Google docs tracks changes, so even in the event of an anonymous vandal, the history is available.

**Comment by Aneurysm9** [comment]
> I disagree that transcription would need 100% accuracy. If we are relying on people taking notes, those certainly don't have that level of accuracy. We can't even attribute note taking because everyone is anonymous and writable by everyone, so a malicious actor could easily modify the notes to say whatever they want. I think the notes are a useful tool for discussion during the meeting and a nice overview afterwards, but should not be considered the source of truth.

We should not expect meeting minutes to have 100% statement-level accuracy and they should not attempt to do so.  We should make clear by policy that they are intended to capture the sense of the conversation and not be a verbatim record.

To the extent that we're taking notes in documents that are writable by all persons anonymously we should stop that practice.  As an organization we should be at a point where we have sufficient coverage of roles that we can grant write access to SIG maintainers/approvers and require others to use suggestion that are approved by someone with permissions.  Or we should use a system that doesn't rely on Google docs.

**Comment by austinlparker** [comment]
As an FYI --

All recordings on the YouTube account have been made private pending a GC decision on this issue.

**Comment by trask** [comment]
All - please review proposal at #1431, thx
